code/modules/admin/verbs/randomverbs.dm

```diff
@@ -554,8 +553,7 @@ Traitors and the like can also be revived with the previous role mostly intact.
 	set category = "Admin"
 	set name = "Delete"
 
-	if (!holder)
-		to_chat(src, "Only administrators may use this command.")
+	if(!check_rights(R_ADMIN))

```

Comment:
Maintainers see proc below this do I need to add this check to it (and if so how should I do it to make sure it wont break everything?)